src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -681,6 +698,7 @@ def add_plane_segmentation(
     metadata: Optional[dict],
     plane_segmentation_name: Optional[str] = None,
     plane_segmentation_index: Optional[int] = None,  # TODO: to be removed
+    background_plane_segmentation_name: Optional[str] = None,
```

Comment:
Why not have this include somewhere in `metadata`?

Reply:
I see, like a parallel to the main `plane_segmentation_name`

Reply:
But then the implication is `add_plane_segmentation` adds more than one place segmentation? Why not just contain it to be responsible for writing a single table and then write multiple tables by iterating outside of it? That way the complexity exists outside rather than inside the function